apparmor.d/groups/kde/kwin_wayland

```diff
@@ -68,10 +71,19 @@ profile kwin_wayland @{exec_path} flags=(attach_disconnected mediate_deleted) {
   owner /var/lib/sddm/.config/kwinrc.lock rwk,
   owner /var/lib/sddm/.config/kwinrc{,.@{rand6}} rwl -> /var/lib/sddm/.config/#@{int},
 
-  owner @{user_cache_dirs}/{,plasma-svgelements} r,
+  owner @{user_cache_dirs}/#@{int} rw,
+  owner @{user_cache_dirs}/{,plasma-svgelements} r
+  owner @{user_cache_dirs}/fontconfig/*-le64.cache-@{int} w,
+  owner @{user_cache_dirs}/fontconfig/*-le64.cache-@{int}{,TMP-@{rand6},NEW,LCK} w,
```

Comment:
Use `abstractions/fontconfig-cache-write` instead

Reply:
I had looked into this abstraction. But is `le64` covered by `[a-f0-9]*` ?